pkg/controller/persistentvolume/persistentvolume_claim_binder_controller.go

```diff
@@ -281,6 +315,19 @@ func syncClaim(volumeIndex *persistentVolumeOrderedIndex, binderClient binderCli
 		if err != nil {
 			return err
 		}
+
+		// claims requesting provisioning will only match the volume provisioned for it.
+		// Another process is provisioning a volume for the claim.  A race condition exists between this controller
+		// and the one provisioning this claim.  The PV created for the claim might not have been created yet, so
```

Comment:
Expand on this race?  Is it patched up somewhere else?  Is it going to leak PVs that were provisioned for claims that did not end up needing them?